### PR TITLE
MAUI: Update Sample Viewer Screenshots

### PR DESCRIPTION
**Pull request by amweis226**
# Description

Update screenshots using .NET MAUI for the Map, MapView, Scene, SceneView, Search, Symbology, and Security categories.

## Type of change

- [ ] Bug fix
- [ ] New sample implementation
- [x] Sample viewer enhancement
- [ ] Other enhancement

**Platforms tested on**:

- [x] WPF .NET 6
- [ ] WPF Framework
- [ ] WinUI
- [x] Xamarin.Forms Android
- [x] Xamarin.Forms iOS
- [ ] Xamarin.Forms UWP

## Checklist

- [x] Runs and compiles on all active platforms
- [x] Legacy platforms still compile and run (if applicable)
- [x] Branch is up to date with the latest main/v.next
- [x] All merge conflicts have been resolved
- [x] Self-review of changes
- [x] There are no warnings related to changes
- [x] No unrelated changes have been made to any other code or project files
- [ ] `sample_sync.py` runs without making changes
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] Code is commented with correct formatting
- [x] All variable and method names are good and make sense
- [x] There is no leftover commented code
- [x] Screenshots are correct size and display in description tab

### CODE REVIEW

**Review by duffh — CHANGES_REQUESTED** [review]
I believe on Android these screenshots need to be resized to be 600px height.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml [hunk 1]
```diff
@@ -49,6 +49,7 @@
                         Grid.Column="0"
                         Clicked="IsAddingStartingLocations_Clicked"
                         IsEnabled="False"
+                        MinimumHeightRequest="75"
```
Comment (ZackAllen) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml:52]: I suggest setting the grids row heights to "auto" to resolve the cause of the issue.

**Review by ZackAllen — CHANGES_REQUESTED** [review]
- All screenshots need to be resize to 600 height.
- `CreateTerrainSurfaceTilePackage` does not have a loaded elevation in its screenshot.
- `GetElevationAtPoint` does not have a loaded elevation in its screenshot.
- `OpenMobileScenePackage` screenshot is not loaded.
- `OpenScenePortalItem` is not loaded

**Review by duffh — CHANGES_REQUESTED** [review]
Most of these look good but some of them are a little too zoomed out or don't fully show the UI as they did in the previous screenshots. I may be nitpicking but would it be possible to zoom in on the relevant data? 

Zoomed out:
- `AccessLoadStatus`
- `DisplayMap`
- `GenerateOfflineMap`
- `HonorMobileMapPackageExpiration`
- `ManageOperationalLayers`
- `OfflineBasemapByReference`
- `OpenMobileMap`
- `ChangeTimeExtent`
- `FeatureLayerTimeOffset`
- `ShowCallout`
- `TakeScreenshot`
- `GeoViewSync`
- `FindAddress` could use the address in the old sample screenshot.
- `FindLocation` need to use current location or a set location.
- `RenderMultilayerSymbols`

Missing UI:
- `AuthorMap`
- `SearchPortalMaps` missing basemap.
- `DisplayGrid`
- `IdentifyLayers`
- `ShowMagnifier`
- `ConfigureSubnetworkTrace` use data from the old screenshot.

UI tweak:
- `BrowseBuildingFloors` the label and picker at the top of the screen are differently aligned. 

Something looks wrong:
- `ChangeAtmosphereEffect` missing 3D image compared to old screenshot.
- `TokenSecuredChallenge` should take the screenshot after the map has loaded as in the old screenshot.

**Comment by ZackAllen** [comment]
AuthorMap - does not show any UI

SetInitialMapArea is incorrect

Identify layers - show popup result

ShowMagnifier - screenshot needs to show the ShowMagnifier

TakeScreenshot - zooming in on map would help a lot here

ChangeAtmosphereEffect - elevation not working correctly

GetElevationAtPoint - UI cropped for no reason

TokenSecuredChallenge - show the loaded map instead of that login screen

PerformValveIsolationTrace - show the result of tracing "Isolating" with include isolated features checked

**Pull request merged**